packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -701,206 +701,6 @@ void main() {
-          '        IosPlugin.registerWith();\n'
-          '      } catch (err) {\n'
-          '        print(\n'
-          "          '`url_launcher_ios` threw an error: \$err. '\n"
-          "          'The app may not function as expected until you remove this plugin from pubspec.yaml'\n"
-          '        );\n'
-          '        rethrow;\n'
-          '      }\n'
-          '\n'
-          '    } else if (Platform.isLinux) {\n'
-          '      try {\n'
-          '        LinuxPlugin.registerWith();\n'
-          '      } catch (err) {\n'
-          '        print(\n'
-          "          '`url_launcher_linux` threw an error: \$err. '\n"
-          "          'The app may not function as expected until you remove this plugin from pubspec.yaml'\n"
-          '        );\n'
-          '        rethrow;\n'
-          '      }\n'
-          '\n'
-          '    } else if (Platform.isMacOS) {\n'
-          '      try {\n'
-          '        AwesomeMacOS.registerWith();\n'
-          '      } catch (err) {\n'
-          '        print(\n'
-          "          '`awesome_macos` threw an error: \$err. '\n"
-          "          'The app may not function as expected until you remove this plugin from pubspec.yaml'\n"
-          '        );\n'
-          '        rethrow;\n'
-          '      }\n'
-          '\n'
-          '      try {\n'
-          '        MacOSPlugin.registerWith();\n'
-          '      } catch (err) {\n'
-          '        print(\n'
-          "          '`url_launcher_macos` threw an error: \$err. '\n"
-          "          'The app may not function as expected until you remove this plugin from pubspec.yaml'\n"
-          '        );\n'
-          '        rethrow;\n'
-          '      }\n'
-          '\n'
-          '    } else if (Platform.isWindows) {\n'
-          '      try {\n'
-          '        WindowsPlugin.registerWith();\n'
-          '      } catch (err) {\n'
-          '        print(\n'
-          "          '`url_launcher_windows` threw an error: \$err. '\n"
-          "          'The app may not function as expected until you remove this plugin from pubspec.yaml'\n"
-          '        );\n'
-          '        rethrow;\n'
-          '      }\n'
-          '\n'
-          '    }\n'
-          '  }\n'
-          '\n'
-          '}\n'
-          '\n'
-          'typedef _UnaryFunction = dynamic Function(List<String> args);\n'
-          'typedef _NullaryFunction = dynamic Function();\n'
-          '\n'
-          'void main(List<String> args) {\n'
-          '  if (entrypoint.main is _UnaryFunction) {\n'
-          '    (entrypoint.main as _UnaryFunction)(args);\n'
-          '  } else {\n'
-          '    (entrypoint.main as _NullaryFunction)();\n'
-          '  }\n'
-          '}\n',
-        );
-      }, overrides: <Type, Generator>{
-        FileSystem: () => fs,
-        ProcessManager: () => FakeProcessManager.any(),
-      });
-
-      testUsingContext('Rewires entrypoints', () async {
```

Comment:
is this test no longer valid?

Reply:
That's right.  The git diff is goofy but the result is removing one test.  We don't rewire entrypoints anymore.